documentation/sphinx/source/release-notes/release-notes-700.rst

```diff
@@ -9,39 +9,69 @@ Release Notes
 
 Features
 --------
+* First release of the Redwood Storage Engine, a BTree storage engine with higher throughput and lower write amplification than SQLite. See :ref:`Redwood Storage Engine Documentation <redwood-storage-engine>` for details.
+* Replaced committed version broadcast through proxy with centralizing live committed versions into master. `(PR #3307) <https://github.com/apple/foundationdb/pull/3307>`_
 * Added a new API in all bindings that can be used to get a list of split points that will split the given range into (roughly) equally sized chunks. `(PR #3394) <https://github.com/apple/foundationdb/pull/3394>`_
+* Introduced a new role called GRV proxy specialized for serving GRV requests to decrease GRV tail latency since we prioritize commit paths over GRV in the current proxy. The original proxy is renamed to Commit proxy. `(PR #3549) <https://github.com/apple/foundationdb/pull/3549>`_ `(PR #3772) <https://github.com/apple/foundationdb/pull/3772>`_
 * Added support for writing backup files directly to Azure blob storage. This is not yet performance tested on large-scale clusters. `(PR #3961) <https://github.com/apple/foundationdb/pull/3961>`_
+* Tag-based throttling now also takes the write path into account. `(PR #3512) <https://github.com/apple/foundationdb/pull/3512>`_
+* Added the ability to ratekeeper to throttle certain types of tags based on write hot spots in addition to read hot spots. `(PR #3571) <https://github.com/apple/foundationdb/pull/3571>`_
+* Users now have the option to make ratekeeper recommend which transaction tags should be throttled, but not actually throttle them using fdbcli. `(PR #3669) <https://github.com/apple/foundationdb/pull/3669>`_
+* Added a new ``--build_flags option`` to binaries to print build information. `(PR #3769) <https://github.com/apple/foundationdb/pull/3769>`_
+* Added ``--incremental`` option to backup and restore that allows specification of only recording mutation log files and not range files. Incremental restore also allows restoring to a non-empty destination database. `(PR #3676) <https://github.com/apple/foundationdb/pull/3676>`_
+* Added a tracing framework to track request latency throughout the system. See :ref:`Documentation <request-tracing>` for details. `(PR #3329) <https://github.com/apple/foundationdb/pull/3329>`_
+* Added the `(Global Configuration Framework) <https://github.com/apple/foundationdb/wiki/Global-Configuration-Framework>`_, an eventually consistent configuration mechanism to efficiently make runtime changes to all clients and servers. `(PR #4330) <https://github.com/apple/foundationdb/pull/4330>`_
```

Comment:
This documentation link can be updated to the new global configuration page (`global-configuration.rst`) in #5068.

Reply:
ok.